scylla/src/statement/prepared_statement.rs

```diff
@@ -335,6 +346,83 @@ pub enum PartitionKeyError {
     ValueTooLong(usize),
 }
 
+pub(crate) type PartitionKeyValue<'ps> = (&'ps [u8], &'ps ColumnSpec);
+
+pub(crate) struct PartitionKey<'ps> {
+    partitioner_name: &'ps PartitionerName,
+    pk_values: SmallVec<[Option<PartitionKeyValue<'ps>>; PartitionKey::SMALLVEC_ON_STACK_SIZE]>,
+}
```

Comment:
It's a large object that now gets also constructed in a single-value partition key case. Did you re-do benchmarks after introducing it?

Reply:
I did, and it seems that it's even better now than previously.